content/en/docs/guides/deadlines.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Deadlines
+description: >-
+  Explains how deadlines can be used to effectively deal with unreliable
+  backends.
+---
+
+### Overview
+
+A deadline is used to specify how long a client is willing to wait for a server
+to respond. This simple idea is very important in building robust distributed
+systems. Clients that do not wait around unnecessarily and servers that know
+when to give up processing requests will improve the resource utilization and
+latency of your system.
+
+Note that while some language APIs have the concept of a __deadline__, others
+use the idea of a __timeout__. When an API asks for a deadline, you provide a
+point in time which the request should not go past. A timeout is the max
+duration of time that the request can take. For simplicity, we will only refer
+to deadline in this document.
+
+### Deadlines on the Client
+
+The default deadline for a gRPC client is some very large number that would not
```

Comment:
Isn't the default deadline effectively an infinite timeout?

Reply:
This whole sentence was awkward, I rephrased it.